CPAC/pipeline/cpac_pipeline.py

```diff
@@ -887,8 +887,7 @@ def build_anat_preproc_stack(rpool, cfg, pipeline_blocks=None):
                 ]
         elif cfg.anatomical_preproc['acpc_alignment']['acpc_target'] == 'whole-head':
             if (rpool.check_rpool('space-T1w_desc-brain_mask') and \
-                cfg.anatomical_preproc['acpc_alignment']['align_brain_mask']) or \
-                    cfg.surface_analysis['freesurfer']['run_reconall']:
```

Comment:
Woah, was this the problem? I'm impressed you (anyone) could hunt that down so quickly.

Reply:
Steve is a wizard

Reply:
https://i.kym-cdn.com/photos/images/newsfeed/000/117/814/are-you-wizard.jpg